src/client/Microsoft.Identity.Client/AuthScheme/TokenType.cs

```diff
@@ -32,6 +32,12 @@ internal enum TokenType
         /// Extension token type.
         /// Extension = 5
         /// </summary>
-        Extension = 5
+        Extension = 5,
```

Comment:
@gladjohn @trwalke  - can you confirm if the token type is captured in our client side and service side telemetry ?

Reply:
For SNI + MTLS private preview to IDDP the usage is going to be just MSI ENG team, so we are not enabling client side telemetry yet. We do have `tokenbindinginfo` and `clientcredentialtype` captured at Server side. 

![image](https://github.com/user-attachments/assets/60b1cd93-1777-4dd0-aee9-9d0ff7d4b9bd)

![image](https://github.com/user-attachments/assets/20d02ead-3934-4a09-8cd2-8251ba7f5a26)

But we will be validating the server side telemetry once the e2e is fully testable.